.github/workflows/conflict_labeler.yml

```diff
@@ -0,0 +1,24 @@
+name: Merge Conflict Labeler
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request_target:
+  issue_comment:
+
+permissions: {}
+
+jobs:
+  label:
+    name: Labeling
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'Sonarr/Sonarr' }}
```

Comment:
This will prevent it from being used down stream without changes, which makes it a little awkward. Is there a way to check if it's a fork and not run on forks?

Reply:
I don't think so this is the best I found to do it, let me know if you know other way to do it